media-gfx/librecad/files/librecad-2.1.3-CVE-2021-21899.patch

```diff
@@ -0,0 +1,821 @@
+From 6417118874333309aa10c4e59f954c3905a6e8b5 Mon Sep 17 00:00:00 2001
+From: Armin Stebich <home@mail.lordofbikes.de>
```

Comment:
Please add a link to where you got it from and also any relevant Gentoo bugs.

Reply:
fixed